verification/simulator.go

```diff
@@ -234,6 +234,11 @@ func GetSimulatorTargets() []TestTarget {
 			getTestTarget([]string{"AutoInit", "Simulation", "X509", "Csr", "IsCA", "RotateContext", "ExtendTci", "IsSymmetric"}),
 			AllTestCases,
 		},
+		{
+			"CertifyKey_TcbValidation",
+			getTestTarget([]string{"AutoInit", "Simulation", "X509", "Csr", "IsCA", "RotateContext", "ExtendTci"}),
+			[]TestCase{DiceTcbValidationTestCase, DiceTcbValidationSimulationTestCase},
+		},
```

Comment:
Have added this as separate test target. 
As per a comment in #PR250, it was mentioned to move tests that modify DPE tree out of basic testcases.
I addressed it in this PR because it is relevant to CertifyKey command.
If i had to include these tests in "AllTestCases" list, I had to use RetainParent, ChangeLocality(for Sim mode), so that the parent context wouyld be available for further tests.